app/src/main/res/layout/fragment_setting.xml

```diff
@@ -140,12 +140,28 @@
         android:ems="10"
         android:hint="*********"
         android:imeOptions="actionDone"
-        android:singleLine="true"
-        android:lines="1"
         android:inputType="textPassword"
+        android:lines="1"
+        android:singleLine="true"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/tv_setting_length6" />
 
+    <Button
+        android:id="@+id/DeleteAccount"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="88dp"
+        android:backgroundTint="@android:color/holo_red_dark"
+        android:fontFamily="sans-serif"
```

Comment:
I don't think it's necessary to set the font family, as the user usually has already chosen a system default for their device. Maybe just set the text to capital letters to emphasize the button.

Reply:
Done